extensions/azurehybridtoolkit/notebooks/hybridbook/content/hadr/configure-failover.ipynb

```diff
@@ -79,16 +77,12 @@
                 "$ServerSku = \"\"                          # SKU of Servers, must be either \"Enterprise\" or \"Developer\".\r\n",
```

Comment:
I'd remove comments from the variables in the code block as the table above it already explains them. Cleans up the code block a little better